openc3/lib/openc3/utilities/store_queued.rb

```diff
@@ -62,7 +62,7 @@ def initialize(update_interval)
     end
 
     def set_update_interval(interval)
-      if interval < @update_interval and interval > 0.0
+      if interval > 0.0
```

Comment:
The instance method sets the default to 1s so I don't see why we should limit this to < 1 ... actually _increasing_ this value would help with performance.